tool/get_localid

```diff
@@ -1,100 +1,100 @@
-#!/usr/bin/python
```

Comment:
Ha,

Het waren niet de tabs; alle Python lines zijn nu met \r\n versierd, dat gaat me wat te ver als porting effort :-D  Kun je dat aanpassen svp?

Ehm, "ignore" is geen optie voor me, ik accepteer de Pull Request of niet...

-Rick